features/apply_with_dependant_stacks.feature

```diff
@@ -0,0 +1,169 @@
+Feature: Apply command
```

Comment:
Yeah I made that spelling mistake everywhere at first, need to do a grep to find 'em all